media-libs/embree/embree-3.13.0.ebuild

```diff
@@ -7,29 +7,32 @@ inherit cmake flag-o-matic linux-info toolchain-funcs
 
 DESCRIPTION="Collection of high-performance ray tracing kernels"
 HOMEPAGE="https://github.com/embree/embree"
-LICENSE="Apache-2.0"
-KEYWORDS="~amd64 ~x86"
 SRC_URI="https://github.com/embree/embree/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
 SLOT="3"
+KEYWORDS="~amd64 ~x86"
 X86_CPU_FLAGS=( sse2:sse2 sse4_2:sse4_2 avx:avx avx2:avx2 avx512dq:avx512dq )
 CPU_FLAGS=( ${X86_CPU_FLAGS[@]/#/cpu_flags_x86_} )
 IUSE="+compact-polys ispc +raymask ssp +tbb tutorial static-libs ${CPU_FLAGS[@]%:*}"
+RESTRICT="mirror"
+
 BDEPEND="
 	virtual/pkgconfig
 "
 RDEPEND="
 	ispc? ( dev-lang/ispc )
-	>=media-libs/glfw-3.2.1
 	tbb? ( dev-cpp/tbb )
 	tutorial? (
 		>=media-libs/libpng-1.6.34:0=
-		>=media-libs/openimageio-1.8.7
+		>=media-libs/openimageio-1.8.7:0=
 		virtual/jpeg:0
 	)
+	>=media-libs/glfw-3.2.1
 	virtual/opengl
```

Comment:
Those two should go on top of the RDEPEND please, c.f. https://devmanual.gentoo.org/general-concepts/dependencies/

Reply:
Ah, I had a brain fart then I read your previous message.
I did bottom -> top instead of top -> bottom order in my head instead XD